tools/demos/Makefile

```diff
@@ -23,6 +23,11 @@ all-gadgets: $(addsuffix /demo.gif,$(DEMO_LIST_EXCEPT_INSTALL))
 	agg $*/demo.cast $*/demo.gif
 	ls -lh $*/demo.gif
 
+.PHONY: check-terminal
+check-terminal:
+	test `tput lines` -eq 24
```

Comment:
What about using https://stackoverflow.com/a/8308063 to try to set the size before testing it?